resources/assets/components/ReportbackUploader/index.js

```diff
@@ -49,11 +49,11 @@ class ReportbackUploader extends React.Component {
 
     const reportback = {
       media: this.state.media,
-      caption: this.caption.value,
+      text: this.caption.value,
       campaignId: this.props.campaignId,
       campaignRunId: this.props.campaignRunId,
       northstarId: this.props.northstarId,
-      source: this.props.source,
+      source: 'rogue-admin',
```

Comment:
Instead of passing this, we should let the `PostRepository` set the source as the incoming client id.